src/components/Greeting.jsx

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
It's not a good practise to disable linters, kindly resolve the issue that the linter is pointing out. Thank you. 🙏

Note: there is **src/Greetings.jsx** file that perform the same function as this, kindly remove the redundant file. ℹ️